protostar/testing/cheatcodes/example_cheatcode.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any
+
+from typing_extensions import Protocol
+
+from protostar.starknet import Cheatcode
+from protostar.testing.fuzzing.strategy_descriptor import StrategyDescriptor
+from protostar.testing.test_config import TestConfig, TestMode
+
+
+class ExampleCallable(Protocol):
+    def __call__(self, **kwargs: StrategyDescriptor):
+        ...
+
+
+class ExampleCheatcode(Cheatcode):
+    def __init__(
+        self,
+        syscall_dependencies: Cheatcode.SyscallDependencies,
+        test_config: TestConfig,
+    ):
+        super().__init__(syscall_dependencies)
+        self.test_config = test_config
+
+    @property
+    def name(self) -> str:
+        return "example"
+
+    def build(self) -> ExampleCallable:
+        return self.example
+
+    def example(self, *args: Any, **kwargs: Any) -> None:
+        self.test_config.convert_mode_to(TestMode.FUZZ)
+
+        assert bool(args) != bool(
+            kwargs
+        ), "you have to provide either args or kwargs to the example cheatcode"
+        assign_args = args
+        if kwargs:
+            assign_args = kwargs
+        self.test_config.fuzz_examples.append(assign_args)
```

Comment:
```suggestion
        self.test_config.fuzz_examples |= kwargs
```

Reply:
Are you sure about this? Doesn't seem to work, plus if `args` are provided, this code misses them. Doesn't this operator `|=` work only for the pair of objects of the same type? Here, we have `args, kwargs` that are `tuple` and `dict` respectively and `fuzz_examples` is essentially a list.

Reply:
this is strictly connected to my other suggestions to remove `args` entirely here and to use `PythonData` :)

applying everything you get `dict[str, Any]` on both sides of the `|=` operator